concepts/observability-strategy/strategy.md

```diff
@@ -1,54 +1,60 @@
 
-The mandatory parts for a Kyma runtime are the stages that must be happen inside the runtime. With that, the following points are the major goal of the Kyma feature set:
+In order to support integration as a key element, it is mandatory that a Kyma runtime supports the stages that must happen within a runtime itself (instrumentation, collection, delivery). Thus, the following points will be the major goal of the Kyma feature set:
 1. Instrumentation support: Users usually need custom code for the instrumentation of a workload to expose typical signals like logs, traces, and metrics. Kyma can support common aspects of these tasks with guidelines, best practice guides, and ways of auto-instrumentation (like Istio tracing).
 2. Signal collection: The runtime must provide the basic infrastructure to collect the emitted signals (like a unified log collector). If the user has followed the provided guides, that infrastructure should collect the signals instantly. Special cases (like specific protocols not backed by guides and infrastructure) should be supported by simple customizations (plug in a custom trace converter). Furthermore, the infrastructure must already enrich the signals of the workloads with metadata of the infrastructure, because this cannot happen at a later stage.
 3. Signal delivery: The signals must be consumed by some party, either running inside or outside the cluster. It can be an aggregating layer or the backend directly. In either case, the shipment of the data must be configurable and should be based on a neutral protocol, so that any aggregation layer or backend can be integrated.
 
-### Optional feature set
-
-Optionally, Kyma can provide features for the stages to the right (aggregation, storage, and analysis). Here, users usually must pick an opinionated solution because there is no common technology available. Kyma can support such opinionated solutions by pre-configuring either a starting point or a full-blown solution. However, these aspects will usually not cover all usage, operations, and scalability scenarios. Furthermore, huge investments are necessary to provide any additional backend solution.
-
-
-### Goals: Summary
+Optionally, Kyma can provide features for the later stages (aggregation, storage, and analysis). Here, users typically must pick an opinionated solution because there is no common technology available. Kyma can support such opinionated solutions by pre-configuring either a starting point or a full-blown solution. However, these aspects will usually not cover all usage, operations, and scalability scenarios. Furthermore, huge investments are necessary to provide any additional backend solution.
```

Comment:
Why don't you like my headline suggestion?

```suggestion
### Optional features

Optionally, Kyma can provide features for the later stages (aggregation, storage, and analysis). Here, users typically must pick an opinionated solution because there is no common technology available. Kyma can support such opinionated solutions by pre-configuring either a starting point or a full-blown solution. However, these aspects will usually not cover all usage, operations, and scalability scenarios. Furthermore, huge investments are necessary to provide any additional backend solution.
```